CONTRIBUTING.md

```diff
@@ -335,7 +335,28 @@ since the latest release over the hotfixed `master`.
 This would mean, however, that the commits referenced from PRs previously merged to `develop` will no longer be part of `develop`'s history,
 which is rather unacceptable.
 
+### Plugin signing
 
+The valid non-expired key pair together with the password required for the plugin signing should be present in the CI environment.
+If they are absent, please take a look at the [plugin signing in IntelliJ](https://plugins.jetbrains.com/docs/intellij/plugin-signing.html#signing-methods) for updated instructions on how to do it.
+Currently, you would need to first generate the private key with:
+```shell
+openssl genpkey\
```

Comment:
Is it possible for the password to be taken from stdin or env var?

Reply:
maybe`echo $PASSWORD_ENV_VAR | openssl ...`

Reply:
because it is possible to pass it through stdin

Reply:
Okay, let's go for that then

Reply:
As in, pls update the instruction accordingly

Reply:
the `password` being in an env var on a local machine, when generation the key pair?